perl-package/AI-MXNet/lib/AI/MXNet/Symbol/NameManager.pm

```diff
@@ -75,10 +75,10 @@ method get(Maybe[Str] $name, Str $hint)
 
 method current()
 {
-    $AI::MXNet::current_nm_ldr;
+    $mx::NameManager;
 }
 
-$AI::MXNet::current_nm_ldr = __PACKAGE__->new;
+$mx::NameManager = __PACKAGE__->new;
```

Comment:
this variable would be better inside this module's namespace.

Reply:
This var is sometimes exposed to user asking them to alter default naming pattern via
local($mx::NameManager) = mx->name->Prefix("prefix_");
This is consistent with local($mx::AttrScope) = mx->AttrScope(...);
and local($mx::Context) = mx->gpu(1); 
which is in its turn consistent with the python examples.

Reply:
Actually you're right I'll fix that particular usage, good catch.